src/components/TextArea/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import _get from 'lodash/get';
+
+import { Answer } from 'types/answer';
+
+export const textAreaDataTestIds = {
+  base: 'text-area__base',
+};
+
+type TextAreaProps = {
+  items: Answer[];
```

Comment:
As this is a base component, you just need to add the essential attributes e.g. `value`, `placeholder`. It should not be specific to any domain.
You can improve it later

Reply:
@hoangmirs Thank you, I will take note of this.